readme.md

```diff
@@ -52,10 +52,21 @@ let mainWindow;
 })();
 ```
 
+Note that the return value of `contextMenu()` is a dispose function:
```

Comment:
The readme and index.d.ts should be in sync documentation-wise.

Reply:
There are a couple of differences, as I didn't know about the sync aspect. Will adjust.
Actually, there's even a react hook example in index.d.ts that is not in readme. 
I will remove that altogether as it feels out of scope here.

Reply:
> The readme and index.d.ts should be in sync documentation-wise.

Synced